seatunnel-e2e/seatunnel-connector-v2-e2e/connector-assert-e2e/src/test/java/org/apache/seatunnel/e2e/connector/assertion/FakeSourceToAssertIT.java

```diff
@@ -18,14 +18,17 @@
 package org.apache.seatunnel.e2e.connector.assertion;
 
 import org.apache.seatunnel.e2e.common.TestSuiteBase;
+import org.apache.seatunnel.e2e.common.container.EngineType;
 import org.apache.seatunnel.e2e.common.container.TestContainer;
+import org.apache.seatunnel.e2e.common.junit.DisabledOnContainer;
 
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.TestTemplate;
 import org.testcontainers.containers.Container;
 
 import java.io.IOException;
 
+@DisabledOnContainer(value = {}, type = {EngineType.SEATUNNEL, EngineType.FLINK}, disabledReason = "TODO: Transform v2 translation to flink isn't completed")
```

Comment:
revert

Reply:
Do you mean that
@DisabledOnContainer(value = {}, type = {EngineType.SEATUNNEL, EngineType.FLINK, EngineType.SPARK}, disabledReason = "TODO: Transform v2 translation to flink/spark isn't completed")
?

Reply:
I see what you mean, thanks.